contracts/colony/Colony.sol

```diff
@@ -37,6 +38,35 @@ contract Colony is ColonyStorage, PatriciaTreeProofs {
     return token;
   }
 
+  bytes4 constant APPROVE_SIG = bytes4(keccak256("approve(address,uint256)"));
+  bytes4 constant TRANSFER_SIG = bytes4(keccak256("transfer(address,uint256)"));
+
+  function makeArbitraryTransaction(address _to, uint256 _value, bytes memory _action)
```

Comment:
`_value` surely must have to be zero, for the same reason that we have to disable `transfer()` on ERC20s?

Reply:
Alternatively, we could assume it has to come out of the pot associated with the root domain, though we could do that with ERC20s as well.

Reply:
Mm I suppose you're right. In my mind I wanted to allow for arbitrary transactions which require value transfer, but I see now how that just open the can of worms associated with feeless payouts, which we're not ready to handle yet.